### ISSUE TITLE
Cannot compile anything - Getting compile errors from builtin.zig file

### ISSUE THREAD

**Issue by rebb** [ziglang/zig#6199]
I want to try out zig, but can't seem to compile even the simplest zig files.
The hello-world example does not compile, neither does something that just has a main() function and nothing else in it.

The compile errors seem to come from a builtin zig file, and not my own code.

This is on windows, using the precompiled package ( fully unpacked etc. ).

I tried deleting the \stage1 directory a few times, after which zig rebuilds it. It does not help.

Here's some of the top errors, all errors are of this type :

```
\AppData\Local\zig\stage1\builtin\Ieuzv1ibvvrdS-vGns50iXNHlL_k5glsKBK8YqGZwrb9tDggV3frMvESUCyOw9j8\builtin.zig:1:1: error: import of 'endian' overrides existing definition
usingnamespace @import("std").builtin;
^
\AppData\Local\zig\stage1\builtin\Ieuzv1ibvvrdS-vGns50iXNHlL_k5glsKBK8YqGZwrb9tDggV3frMvESUCyOw9j8\builtin.zig:3:5: note: previous definition here
pub const endian = Endian.Little;
    ^
\AppData\Local\zig\stage1\builtin\Ieuzv1ibvvrdS-vGns50iXNHlL_k5glsKBK8YqGZwrb9tDggV3frMvESUCyOw9j8\builtin.zig:3:5: note: imported definition here
pub const endian = Endian.Little;
    ^
\AppData\Local\zig\stage1\builtin\Ieuzv1ibvvrdS-vGns50iXNHlL_k5glsKBK8YqGZwrb9tDggV3frMvESUCyOw9j8\builtin.zig:1:1: error: import of 'output_mode' overrides existing definition
usingnamespace @import("std").builtin;
^
\AppData\Local\zig\stage1\builtin\Ieuzv1ibvvrdS-vGns50iXNHlL_k5glsKBK8YqGZwrb9tDggV3frMvESUCyOw9j8\builtin.zig:4:5: note: previous definition here
pub const output_mode = OutputMode.Exe;
    ^
\AppData\Local\zig\stage1\builtin\Ieuzv1ibvvrdS-vGns50iXNHlL_k5glsKBK8YqGZwrb9tDggV3frMvESUCyOw9j8\builtin.zig:4:5: note: imported definition here
pub const output_mode = OutputMode.Exe;
    ^
\AppData\Local\zig\stage1\builtin\Ieuzv1ibvvrdS-vGns50iXNHlL_k5glsKBK8YqGZwrb9tDggV3frMvESUCyOw9j8\builtin.zig:1:1: error: import of 'link_mode'overrides existing definition
usingnamespace @import("std").builtin;
^
\AppData\Local\zig\stage1\builtin\Ieuzv1ibvvrdS-vGns50iXNHlL_k5glsKBK8YqGZwrb9tDggV3frMvESUCyOw9j8\builtin.zig:5:5: note: previous definition here
pub const link_mode = LinkMode.Static;
    ^
\AppData\Local\zig\stage1\builtin\Ieuzv1ibvvrdS-vGns50iXNHlL_k5glsKBK8YqGZwrb9tDggV3frMvESUCyOw9j8\builtin.zig:5:5: note: imported definition here
pub const link_mode = LinkMode.Static;
    ^
```

**Comment by rebb** [comment]
One thing to note maybe, my UserData directory is a Windows Junction, so i can have my UserData stuff on a different partition.

**Comment by andrewrk** [comment]
I believe this was a problem with the caching system on Windows that is now fixed after the merge of #6250.

**Comment by rebb** [comment]
Thank you :) Unfortunately, a new issue has cropped up in it's place now. https://github.com/ziglang/zig/issues/6608
